fastai/data/external.py

```diff
@@ -110,15 +112,24 @@ class URLs():
     WT103_FWD          = f'{S3_MODEL}wt103-fwd.tgz'
     WT103_BWD          = f'{S3_MODEL}wt103-bwd.tgz'
 
-    def path(url='.', c_key='archive'):
+    def path(
+        url='.', # URL of file to download
```

Comment:
You could remove "URL of" here, since that's already implied by the param name

Reply:
Done